> ValueError: License file given in about/license_file (/home/travis/build/SymbiFlow/conda-packages/sv-parser/LICENSE-MIT LICENSE-APACHE) does not exist in source root dir or in recipe root dir (with meta.yaml)


```
Packaging sv-parser
INFO:conda_build.build:Packaging sv-parser
INFO conda_build.build:build(1566): Packaging sv-parser
Packaging sv-parser-33eff2dd93fd74799e3e06453235fa2c800c40cd-20191023_074245
INFO:conda_build.build:Packaging sv-parser-33eff2dd93fd74799e3e06453235fa2c800c40cd-20191023_074245
INFO conda_build.build:bundle_conda(886): Packaging sv-parser-33eff2dd93fd74799e3e06453235fa2c800c40cd-20191023_074245
number of files: 1
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: /lib64/ld-linux-x86-64.so.2
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: $RPATH/libdl.so.2
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: $RPATH/libc.so.6
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: $RPATH/librt.so.1
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: $RPATH/libpthread.so.0
WARNING (sv-parser,bin/parse_sv): did not find - or even know where to look for: $RPATH/libgcc_s.so.1
Fixing permissions
Traceback (most recent call last):
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 449, in main
    execute(sys.argv[1:])
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 440, in execute
    verify=args.verify, variants=args.variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 2339, in build_tree
    notest=notest,
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 1638, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 1010, in bundle_conda
    output['checksums'] = create_info_files(metadata, files, prefix=metadata.config.host_prefix)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 704, in create_info_files
    copy_license(m)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 398, in copy_license
    "source root dir or in recipe root dir (with meta.yaml)".format(src_file))
ValueError: License file given in about/license_file (/home/travis/build/SymbiFlow/conda-packages/sv-parser/LICENSE-MIT LICENSE-APACHE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -46,6 +46,7 @@ env:
   - PACKAGE=tree-sitter-verilog
   - PACKAGE=slang               CONDA_CHANNELS="conda-forge"
   - PACKAGE=moore
+  - PACKAGE=sv-parser
   # protocol analyzers
   - PACKAGE=sigrok-cli          CONDA_CHANNELS="conda-forge"
 

--- a/sv-parser/build.sh
+++ b/sv-parser/build.sh
@@ -1,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+source $HOME/.cargo/env
+
+cargo build --example parse_sv --release
+install -D target/release/examples/parse_sv $PREFIX/bin/parse_sv
+
+$PREFIX/bin/parse_sv --version

--- a/sv-parser/meta.yaml
+++ b/sv-parser/meta.yaml
@@ -1,0 +1,26 @@
+package:
+  name: sv-parser
+  version: {{ GIT_FULL_HASH }}
+
+source:
+  git_url: https://github.com/dalance/sv-parser
+  git_rev: master
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+test:
+  commands:
+    - parse_sv --version
+
+about:
+  home: https://github.com/dalance/sv-parser
+  license: MIT
+  license_file: LICENSE-MIT
+  summary: SystemVerilog parser library fully complient with IEEE 1800-2017

